appuio/haproxy/templates/configmap-galera-checkscript.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if or (eq .Values.haproxy.config "galera") (eq .Values.haproxy.config "galerak8s") }}
+
+kind: ConfigMap
+apiVersion: v1
+data:
+  script.sh: "#!/bin/bash\n\n# Run the MySQL query and store the result in a Bash
+    variable\nwsrep_ready=$(/bin/cat /secrets/mariadb-root-password | /usr/bin/mysql
+    -h $3 -u root -p -e \"SELECT VARIABLE_VALUE FROM information_schema.global_status
```

Comment:
Instead of using `cat`you can mount the secret as env variable and pass it to the mysql command like this:
```
mysql -h $3 -u root -p $MARIADB_ROOT_PASSWORD -e [...]
```

Reply:
because script does not have access to env variables, therefore it must be attached as file :/ script is run as subprocess of haproxy and haproxy does not pass envs into it